src/main/java/com/fasterxml/jackson/databind/ser/BeanSerializer.java

```diff
@@ -126,6 +126,11 @@ protected BeanSerializerBase withProperties(BeanPropertyWriter[] properties,
         return new BeanSerializer(this, properties, filteredProperties);
     }
 
+    @Override // @since 2.16
+    public JsonSerializer<?> withIgnoredProperties(Set<String> toIgnore) {
+        return new BeanSerializer(this, toIgnore, null);
```

Comment:
My only concern here is that passing `null` could remove effects of possible `@JsonIncludeProperties`. But looking at `BeanSerializerBase`, this information is not really retained explicitly -- but hopefully `src._filteredProps` has information.

So this is why having those tests is good verification.

Reply:
Sorry, no, `null` is fine (wrt empty). I was thinking out aloud that originally provided inclusion cannot be passed again (which would come from `@JsonIncludeProperties`).

But let's worry about this if someone can figure out case where this is problematic.